pkgs/flox-tests/default.nix

```diff
@@ -57,6 +58,7 @@ in
   writeShellScriptBin "flox-tests" ''
 
         export PATH="${lib.makeBinPath paths}"
+        export PKGDB_BIN="${flox.passthru.envs.PKGDB_BIN}"
```

Comment:
Shouldn't this be encoded at build time already?

Reply:
PKGDB_BIN is compiled into the flox binary, but I'm directly invoking pkgdb lock, not using the flox binary at all

Reply:
OH gotcha.

In that case you don't need `passthru`.
Simply add `pkgdb` to the arguments at the top and consume the package directly.

Our overlays ensure that only a single build of `pkgdb` is available in the closure, so there's no need to stash anything away in `passthru` attributes or anything like that anymore.